src/App.js

```diff
@@ -24,17 +27,37 @@ class App extends Component {
     });
   }
 
+  getForecast(city){
+    fetchForecast(city)
+    .then((response) => {
+      if (this.refs.myRef) {
+        this.setState({
+          forecast: response,
+        });
+      }
+      console.log(response);
+    });  
+  }
+
   handleChange(city){
-    this.getWeather(city);    
+    this.getWeather(city);
+    this.getForecast(city);    
+  }
+
+  handleClick(newWeather){
+    this.setState({
+      weather: newWeather
+    });
   }
 
   render() {
-    if(this.state.weather)   
+    if(this.state.weather && this.state.forecast)    
     return (
       <Container textAlign='center'>
         <div className="App" ref="myRef">
           <Search onChange={(city) => this.handleChange(city)} />
-          <WeatherItem currentWeather={this.state.weather} />
+          <WeatherItem currentWeather={this.state.weather} /> 
+          <WeatherWeek forecast={this.state.forecast} onClick={(newWeather) => this.handleClick(newWeather)}/>         
```

Comment:
what would be the idea of wrapping `this.handleClick` to another function?
```
function (newWeather) {
  this.handleClick(newWeather);
}
```
should this just be `onClick={this.handleClick}/>`
?